### ISSUE TITLE
Secondary references

### ISSUE THREAD

**Issue by hughkf** [uwhpsc-2016/lectures#32]
Will the final exam cover the material in the secondary references?

**Comment by cswiercz** [comment]
No. Just like with the quizzes you will only be tested on the material in class and in the primary references.

There may be some questions related to the homework assignments, as well.
